helix-term/src/commands/typed.rs

```diff
@@ -1479,6 +1479,42 @@ fn get_option(
     Ok(())
 }
 
+/// Change config at runtime. Access nested values by dot syntax, for
+/// example to toggle smart case search, use `:toggle search.smart-case`.
+fn toggle_option(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
```

Comment:
any reason for using ```return Ok(());``` instead of `Ok()`

Reply:
using `return` avoids wrapping the rest of the function in an `else` block.
also, it's a matter of being consistent with `set_option` which has the same structure.